libs/ledger-live-common/src/families/ethereum/synchronisation.ts

```diff
@@ -277,11 +277,11 @@ const txToOps =
               };
             }
           })
-          .filter(Boolean) as Operation[]);
+          .filter(Boolean) as unknown as Operation[]);
```

Comment:
I think it we could do the `filter` before the `map` that would make sense

Reply:
No we can't, because we are filtering falsy values from the map